docs/source/notebooks/OGCAPI_processes_example.ipynb

```diff
@@ -0,0 +1,544 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## PAVICS Web Processing Services using OGC-API integration with Weaver\n",
+    "\n",
+    "When [Weaver component](https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/components#weaver)\n",
+    "is enabled, all WPS *birds* registered as process *providers* will be automatically accessible using\n",
+    "[OGC-API - Processes][ogcapi-proc] interface from the endpoint where [Weaver][weaver] is defined.\n",
+    "\n",
+    "[weaver]: https://github.com/crim-ca/weaver\n",
+    "[ogcapi-proc]: https://github.com/opengeospatial/ogcapi-processes/\n",
+    "\n",
+    "**NOTE**:\n",
+    "\n",
+    "> This test will only work when the component is enabled, and therefore requires explicit specification of test\n",
+    "  execution using `WEAVER_TEST_ENABLED = true`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Variables:\n",
+      "  WEAVER_TEST_FQDN         pavics.ouranos.ca\n",
+      "  WEAVER_TEST_URL          http://localhost:4002\n",
+      "  WEAVER_TEST_FILE         https://host-140-4.rdext.crim.ca/twitcher/ows/proxy/thredds/dodsC/birdhouse/nrcan/nrcan_canada_daily/tasmin/nrcan_canada_daily_tasmin_2013.nc\n",
```

Comment:
Shouldn't we take a file from thredds' test dataset located on each platform instance?

Reply:
Yes. If `WEAVER_TEST_FILE` is not specified, it defaults to `https://${WEAVER_TEST_FQDN}//twitcher/ows/proxy/thredds/dodsC/birdhouse/nrcan/nrcan_canada_daily/tasmin/nrcan_canada_daily_tasmin_2013.nc`

In this test run, I simply overwrote the values such that I target my localhost Weaver because the require PR is not yet merged, but both the file and the hummingbird process are on a daccs instance.